Makefile

```diff
@@ -39,6 +39,7 @@ moveout: $(ITEMS)
 	@echo '** Making build directory: $(type) **'
 	cp -r $(ITEMS) build/$(browser)/$(type)
 	cp -r $(ITEMS) build/$(browser)/$(type)
+	cp -r shared/js/content-scripts build/$(browser)/$(type)/public/js/
```

Comment:
If you run this with a deleted `build` dir, it fails. I think we need to run `mkdir -p build/$(browser)/$(type)/public/js/`
```
** Making build directory: dev **
cp -r shared/html shared/data shared/img build/chrome/dev
cp -r shared/html shared/data shared/img build/chrome/dev
cp -r shared/js/content-scripts build/chrome/dev/public/js/
cp: cannot create directory 'build/chrome/dev/public/js/': No such file or directory
Makefile:39: recipe for target 'moveout' failed
```

Reply:
Actually, I'm seeing that without deleting my `build/` dir too.